.github/workflows/publish.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: publish
+on:
+  workflow_run:
+    workflows:
+      - helm
+      - integration-tests
```

Comment:
this will trigger one time for each

Reply:
Yes that's expected. One publish stage for libraries charts (first), then one for for application charts (second, as they most likely depend on library)